src/app/js/admin/parsing/ParsingExcerptColumn.js

```diff
@@ -21,7 +21,7 @@ export const ParsingExcerptColumnComponent = ({ children, style, value }) =>
         </TableCell>
     ) : (
         <TableCell style={{ ...styles.col, ...style }}>
-            {value}
+            {`${value}`}
```

Comment:
Is this a cast to string?

Reply:
Yes, it allows us to display if a value is false